tests/devices/qubit/test_apply_operation.py

```diff
@@ -292,7 +285,7 @@ def test_parameterized_evolution_time_independent(self, method):
 
         H = time_independent_hamiltonian()
         params = jnp.array([1.0, 2.0])
-        t = 4
+        t = 0.4
```

Comment:
Changed a few evolution times by one order of magnitude. Decreased testing time for `ParametrizedEvolution` in this file from 15 to 4.5 seconds